### ISSUE TITLE
Allow style-inject to insert at various places in <head>

### ISSUE THREAD

**Issue by kdar** [egoist/rollup-plugin-postcss#52]
Just like https://github.com/webpack-contrib/style-loader allows you to do with insertAt. This is useful for me as I need the css of this component to be low priority (injecting into the top of head).

**Comment by kdar** [comment]
For the time being, I am working around this by overriding the plugin's intro():

```
import postcss from 'rollup-plugin-postcss';

function styleInject(css, returnValue) {
  if (typeof document === 'undefined') {
    return returnValue;
  }
  css = css || '';
  var head = document.head || document.getElementsByTagName('head')[0];
  var style = document.createElement('style');
  style.type = 'text/css';
  
  var insertBefore = head.children[head.children.length];
  for (var i = 0; i < head.children.length; i++) {
    var tag = head.children[i].tagName.toLowerCase();
    if (tag == "style" || tag == "link") {
      insertBefore = head.children[i];
      break;
    }
  }
  head.insertBefore(style, insertBefore);
  
  if (style.styleSheet){
    style.styleSheet.cssText = css;
  } else {
    style.appendChild(document.createTextNode(css));
  }
  return returnValue;
};

const postcssPlugin = postcss({
  extensions: ['.css'] 
});

postcssPlugin.intro = function() {
  return styleInject
  .toString()
  .replace(/styleInject/, '__$styleInject')
};

export default {
  input: 'src/index.js',
  plugins: [
    postcssPlugin
  ]
}
```

**Comment by t4t5** [comment]
I submitted a [pull request](https://github.com/egoist/style-inject/pull/4) to the `style-inject` library (which is what this plugin uses under the hood) about this. It's been merged and the library was updated to 0.2.0 just now.

Hopefully, that means this feature will arrive here soon as well. :)

**Comment by lmihaidaniel** [comment]
this feature is now implemented #55
